ons_alpha/jinja2/templates/pages/bulletin_page.html

```diff
@@ -122,11 +124,11 @@ <h2 class="heading-2">{{ _("Contact details") }}</h2>
                                 <strong>Name</strong>: {{ page.contact_details.name }}
                             </p>
                             <p class="contact-details__line">
-                                <strong>Email</strong>: {{ page.contact_details.email }}
+                                <strong>Email</strong>: <a href="mailto:{{ page.contact_details.email }}">{{ page.contact_details.email }}</a>
                             </p>
                             {% if page.contact_details.phone %}
                                 <p class="contact-details__line">
-                                    <strong>Telephone</strong>: {{ page.contact_details.phone }}
+                                    <strong>Telephone</strong>: <a href="tel:{{ page.contact_details.phone }}">{{ page.contact_details.phone }}</a>
```

Comment:
Valid tel links can't have spaces in them, so they need to be removed somehow. Does this work in Jinja?
```suggestion
                                    <strong>Telephone</strong>: <a href="tel:{{ page.contact_details.phone|cut:" " }}">{{ page.contact_details.phone }}</a>
```

Reply:
Ah good spot.